accounting_reporting_core/src/main/java/org/cardanofoundation/lob/app/accounting_reporting_core/resource/presentation_layer_service/ReportViewService.java

```diff
@@ -53,6 +53,9 @@ public ReportView responseView(ReportEntity reportEntity) {
         reportResponseView.setDate(reportEntity.getDate());
         reportResponseView.setPublish(reportEntity.getLedgerDispatchApproved());
         reportResponseView.setCanBePublish(true);
+        if (null != reportEntity.getLedgerDispatchReceipt()) {
```

Comment:
Yoda notation again? I would also prefer if this would be returned as Optional<LedgerDispatchReceipt>.

A typical issue in the projects is that over time nobody knows what is null and what note and everybody programs defensively and code turns into one giant mess.

Reply:
why is this resolved?